RFCs/0025-graph-data-model.md

```diff
@@ -54,51 +63,99 @@ Which is similarly extracted as variables:
 SELECT i, v FROM my_list AS v AT i
 ```
 
-For the graph data type, we model something very similar. A graph is a collection of *vertices* and *edges* that connect
-them with associated direction (which may be un-directed). All vertices and edges have a *label* (similar to the
-attribute name in struct) that must be a string or `NULL`. The value of a vertex or an edge can be any PartiQL value.
-The following diagram illustrates the model:
+For the graph data type, we model something very similar. A graph is a collection of *vertices* and *edges*, 
+where edges connect pairs of vertices and can be directed or undirected.
+Vertices and edges have *labels* (similar to the attribute names in struct). 
```

Comment:
I think we should have some clarifying text around what this actually means.  I think it is different from the `struct` example as there is a 1-to-1 correspondence there between attribute names and values.

Reply:
A PartiQL struct does allow repeating attribute names (with different values at them), doesn't it?  So, intuitively, these settings are about the same, in my view: neither a struct's attribute name nor a graph's label are a perfect "address".  Of course, repeating an attribute in a struct seem to be a discouraged corner case, while repeating labels in a property graph is more like the norm.    
Overall, I am not sure what can be said here without getting into the weeds...

Another wart here is that we (as well as GPML) allow multiple labels on a node/edge. I intentionally wrote line 68 to glide past this matter, since this is an intro/intuition text.